admin/src/User/User.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Create,
+  Edit,
+  SimpleForm,
+  TextInput,
+  required,
+  List,
+  Datagrid,
+  TextField,
+  EmailField,
+} from "react-admin";
+
+export const UserList = (props: any) => (
```

Comment:
For now it's ok as the file is not so big. but you could organize your folders to split the components 
```
User/
├─ List.js
├─ Edit.js
├─ index.js
```

Reply:
Thanks for the tip !